
>>> Default:

The error message
Call `rlang::last_error()` to see a backtrace


>>> Reminder:

The error message
Call `rlang::last_error()` to see a backtrace


>>> Branch:

The error message
Backtrace:
 ─expect_known_output(...)
 ─msg()
 ─f()
 ─g()
 ─h()


>>> Collapsed:

The error message
Backtrace:
█
├─[ expect_known_output(...) ] with 5 more calls
├─[ with_options(...) ]
└─msg()
  ├─[ cat(...) ]
  ├─[ conditionMessage(...) ]
  ├─[ catch_cnd(...) ] with 5 more calls
  └─f()
    ├─[ tryCatch(...) ] with 1 more call
    └─g()
      └─h()
